RentACarTests/ControllersTests/ClientControllerTests.cs

```diff
@@ -0,0 +1,254 @@
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using RentACarWebAPI.Controllers;
+using RentACarWebAPI.DTOs;
+using RentACarWebAPI.Interfaces.Services;
+using RentACarWebAPI.Models;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace RentACarTests.ControllersTests
+{
+    public class ClientControllerTests
+    {
+        #region GetAllAsync
+        [Fact]
+        public async Task GetAllAsync_ReturnsClientsDto()
+        {
+            //arrange
+            var clientOneName = "Pedro";
+
+            var client1 = new Client() { Id = 1, Name = clientOneName, Dni = "13131311" };
+            var client2 = new Client() { Id = 3, Name = "Juanse", Dni = "13131313" };
+            var client3 = new Client() { Id = 7, Name = "Sofia", Dni = "89444666" };
+
+            var clientList = new List<Client>
+            {
+                client1,
+                client2,
+                client3
+            };
+
+            var mockClientService = new Mock<IClientService>();
+            mockClientService.Setup(serv => serv.GetAllAsync()).ReturnsAsync(clientList);
+
+            var clientController = new ClientController(mockClientService.Object);
```

Comment:
this applies to both test classes, I believe you can reutilize this piece of code (except the mock setup) by creating the instance of the mocks and the class to be tested in the test class constructor, let me know if you need some help or guidance

Reply:
Oh, that is true. I'll make that change and if I get stuck I'll let you know. thanks!